<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicMobileSongsManagerWidget</class>
 <widget class="QWidget" name="MusicMobileSongsManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>移动歌曲管理</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>760</width>
     <height>550</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>4</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>移动歌曲管理</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>743</x>
     <y>6</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QWidget" name="toolWidget" native="true">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>29</y>
     <width>760</width>
     <height>525</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,150)</string>
   </property>
   <widget class="QCheckBox" name="allSelectedcheckBox">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>12</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="auditionButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>7</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>7</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchLineEdit">
    <property name="geometry">
     <rect>
      <x>246</x>
      <y>7</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#6495ED;border: 1px solid #555;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchLineLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>12</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/tiny/btn_search_main_hover);
background:rgba(255,255,255,50)</string>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="backgroundMask1" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>760</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(255,255,255,75)</string>
    </property>
   </widget>
   <widget class="MusicMobileSongsTableWidget" name="songlistTable">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>760</width>
      <height>485</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="backgroundMask2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>760</width>
      <height>485</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(255,255,255,150)</string>
    </property>
   </widget>
   <zorder>backgroundMask1</zorder>
   <zorder>allSelectedcheckBox</zorder>
   <zorder>auditionButton</zorder>
   <zorder>addButton</zorder>
   <zorder>searchLineEdit</zorder>
   <zorder>searchLineLabel</zorder>
   <zorder>backgroundMask2</zorder>
   <zorder>songlistTable</zorder>
  </widget>
  <widget class="MusicGifLabelWidget" name="loadingLabel">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>270</y>
     <width>58</width>
     <height>58</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicGifLabelWidget</class>
   <extends>QLabel</extends>
   <header>musicgiflabelwidget.h</header>
  </customwidget>
  <customwidget>
   <class>MusicMobileSongsTableWidget</class>
   <extends>QTableWidget</extends>
   <header>musicmobilesongsmanagerwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
